jax/_src/clusters/cluster.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) 2022, NVIDIA CORPORATION. All rights reserved.
```

Comment:
@skye should this be NVIDIA corporation or Google LLC?

Reply:
@hawkinsp

Reply:
The rules we follow here are:
https://opensource.google/documentation/reference/releasing/contributions#copyright

(I think we'd be very open to changing the copyright attribution to "The JAX authors" but we'd have to do that for the entire project.)

Reply:
Sounds good, I just pushed a commit to follow your guideline. But yes if we don't want individual company copyright, it would make sense to switch to "The JAX authors" to match the legal reality of the files copyright.